maro/cli/grass/job.py

```diff
@@ -15,7 +15,7 @@ def start_job(cluster_name: str, deployment_path: str, **kwargs):
     # Load details
     cluster_details = load_cluster_details(cluster_name=cluster_name)
 
-    if cluster_details["mode"] == "grass/azure":
+    if cluster_details["mode"] == "grass/azure" or "grass/on-premises":
```

Comment:
it should be 
if cluster_details["mode"] in ["grass/azure", "grass/on-premises"]: